src/main/python/calibration/generate_rides_with_google_maps.py

```diff
@@ -0,0 +1,237 @@
+#!/usr/bin/env python
+"""Generate A CSV file containing Google Maps links and apply filters from the Beam files: {iteration}.personal.CarRideStats.csv.gz
+Example of usage:
+python generate_rides_with_google_maps.py \
+--inputFile="/folder-path/it.0/0.CarRideStats.csv.gz" \
+--sampleSize=5 \
+--sampleSeed=12345 \
+--travelDistanceIntervalInMeters=[4450,6000] \
+--departureTimeIntervalInSeconds=[25246,25248] \
+--areaBoundBox=[(10,11.1),(12,14)]
+
+Only inputFile is a mandatory parameter. All others are optional
+"""
+
+import pandas as pd
+import getopt, sys
+from pathlib import Path
+import glob
+import random
+import numpy as np
+from tabulate import tabulate
+from urllib.parse import urlparse
+import re
+
+
+__author__ = "Carlos Caldas"
+
+
+def main(args):
```

Comment:
What is the verison of Python? Is it 2 or 3? Anyway, can you use https://docs.python.org/2/library/argparse.html to parse arguments instead of doing a lot of manual work?

Reply:
It is Python 3. It is using `getOpt`. There is no lot of manual work, only default values definitions, so the arguments must be tested.